integration-test/src/test/java/org/apache/iotdb/pipe/it/autocreate/IoTDBPipeAlterIT.java

```diff
@@ -53,7 +53,7 @@ public void testBasicAlterPipe() throws Exception {
     // Create pipe
     final String sql =
         String.format(
-            "create pipe a2b with processor ('processor'='do-nothing-processor') with sink ('node-urls'='%s')",
+            "create pipe a2b with source ('source'='iotdb-source', 'source.pattern'='root.test1', 'source.path'='root.test2.**') with processor ('processor'='do-nothing-processor') with sink ('node-urls'='%s')",
```

Comment:
Typically a pipe won't have "pattern" and "path" parameters together. The "path" one can be omitted here. And we can modify "pattern" instead of "path" too.